.github/workflows/ci.yml

```diff
@@ -70,6 +70,9 @@ jobs:
             tari-${{ runner.os }}-${{ runner.cpu-model }}-${{ env.toolchain }}-nightly
       - name: cargo format
         run: cargo fmt --all -- --check
+      - run: rustup update
+      - run: rustup default nightly-2024-07-07
+      - run: rustup update
```

Comment:
As this is a hack, I would rather suggest replacing the above snippet with the following
```yaml
      - name: rust update hack
      - run: |
           rustup update
           rustup default ${{ env.toolchain }}
           rustup update

      - name: Install cargo-lints
```